### DISCUSSION TITLE
[NETWORK] Upload go up, download go down

### DISCUSSION THREAD

**Discussion by VA1DER** [aristocratos/btop#532] [Ideas]
This is a small thing, but I would like to see on the network activity graph for the upload to be up, and the download to be down.  Currently it is opposite.

**Comment by hugalafutro** [comment]
i would love this too

it always baffles me that in about 3/4 of apps that display ul/dl (like rainmeter, conky etc) the default is download is at the top and upload at the bottom, at least in my experience

think about it, read the words: DOWNload is up, and UPload is down. why? surely it's more intuitive if down is down and up is up no? i don't think i have ocd or w/e, but it always triggers me (if that's the right word lol english ain't my 1st lang.)

**Comment by tonydero** [comment]
I think the perspective of download on top and upload on the bottom is that the top is what's coming in, and the bottom is what's going out, maybe?

**Reply by hugalafutro** [reply]
That makes sense, but at this point I got it ingrained the other way. I also play games with invert mouse if that matters :D An option to toggle the order would appease everyone.

**Reply by tonydero** [reply]
Agreed! Options are always best.
